packages/docs-site/src/components/presenters/text-card/text-card.scss

```diff
@@ -0,0 +1,78 @@
+@import "@royalnavy/css-framework";
+
+$border-radius: 4px;
+$border-color: #dae2ec;
```

Comment:
Are these in the standard colour scheme?

Reply:
Hmm not sure - I need to talk to design on Monday. I've color picked everything.